resources/views/discord/new.blade.php

```diff
@@ -45,7 +45,7 @@
                                 </a>
                             @else
                                 <a href="#" style="text-decoration: none;">
-                                    <button class="btn btn-primary center-block" disabled><em class="fab fa-discord"></em> &thinsp; Discord Account {{ $_account->discord_id }} Linked</button>
+                                    <button class="btn btn-primary center-block" disabled><em class="fab fa-discord"></em> &thinsp; Discord Account {{ $_account->discord_user ? $_account->discord_user['username'].'#'.$_account->discord_user['discriminator'] : $_account->discord_id }} Linked</button>
```

Comment:
You can rely on `$_account->discord_user` returning an correct here as it is within the else statement from line 42. `discord_user` only returns `null` if `discord_id` is blank looking at the getter on the model.

Reply:
Good catch -https://github.com/VATSIM-UK/core/pull/2029/commits/bacf74390e917a543d3d37b3513e289f79bdac1a